docs/Agent/installation/ansible_based_installation.md

```diff
@@ -1,20 +1,16 @@
 # Ansible based installation
 
-In the following, we describe how to install `pbench-agent` using an ansible playbook.
+In the following, we describe how to install Pbench Agent` using an ansible playbook.
```

Comment:
Thanks; but you left the trailing "`" here, which I think will put the rest of the document (or at least the rest of the paragraph) in fixed font!

Reply:
fixed